omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ConceptResource1_8Test.java

```diff
@@ -13,9 +13,17 @@
  */
 package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_8;
 
+import static org.junit.Assert.*;

```

Comment:
We generally discourage wild card imports https://wiki.openmrs.org/display/docs/Coding+Conventions